lib/control/validation/content.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const HTTP_BAD_REQUEST = 400;
+
+const M_INSTANCE_ID = /^i-[a-f0-9]{8,}$/;
```

Comment:
Do these regexes still work when Amazon switches to the new longer ID format? Or will they have to be updated?